sdk/proto/generate.sh

```diff
@@ -40,6 +40,13 @@ $PROTOC --js_out=$JS_PROTOFLAGS:$JS_PULUMIRPC --grpc_out=$JS_PULUMIRPC --plugin=
 PY_PULUMIRPC=../python/lib/pulumi/runtime/proto/
 echo -e "\tPython: $PY_PULUMIRPC"
 mkdir -p $PY_PULUMIRPC
-python -m grpc_tools.protoc -I./ --python_out=$PY_PULUMIRPC --grpc_python_out=$PY_PULUMIRPC *.proto
+
+
+TEMP_DIR=$(mktemp -d)
+echo -e "\tPython temp dir: $TEMP_DIR"
+python -m grpc_tools.protoc -I./ --python_out=$TEMP_DIR --grpc_python_out=$TEMP_DIR *.proto
```

Comment:
Always quote shell variables and glob expansions:
`"$TEMP_DIR"`
`"*.proto"`

Reply:
If I'm globbing, I can't quote, right? I'm using the shell to expand the glob and not protoc.

Reply:
Yes. That latter advice was misguided -- sorry.

```
$ touch "a.txt" "z.txt" "this name has spaces in it.txt"
$ bash -c 'echo $0; echo $1; echo $2' ./*.txt
./a.txt
./this name has spaces in it.txt
./z.txt
```